### DISCUSSION TITLE
How to add anchors to H1-6

### DISCUSSION THREAD

**Discussion by LALBJ** [remarkjs/remark#923] [Q&A] [answered]
I am just a beiginer of react-markdown.
I want to generate an Anchor for H<1-6> to implement my custom TOC jump.
And I see that ‘heading’ has been removed from components.
So the way I'm going to write it is:
<ReactMarkdown
          children={content}
          components={{
            h1: TOC,
            h2: TOC,
            h3: TOC,
          }}
></ReactMarkdown>
Is there a better way to solve this problem？

**Comment by ChristianMurphy** [comment, marked as answer]
It sounds like you'd like headings to automatically have anchors added?
If so https://github.com/rehypejs/rehype-autolink-headings may help.
